include/avif/avif.h

```diff
@@ -602,9 +578,26 @@ AVIF_API uint32_t avifRGBImagePixelSize(const avifRGBImage * rgb);
 AVIF_API void avifRGBImageAllocatePixels(avifRGBImage * rgb);
 AVIF_API void avifRGBImageFreePixels(avifRGBImage * rgb);
 
-// The main conversion functions
-AVIF_API avifResult avifImageRGBToYUV(avifImage * image, const avifRGBImage * rgb);
-AVIF_API avifResult avifImageYUVToRGB(const avifImage * image, avifRGBImage * rgb);
+// The main conversion options.
+typedef enum avifConversionFlag
+{
+    // Conversion between RGB and YUV samples:
+    AVIF_CONVERSION_AUTO = 0,                // prefers libyuv if available (default)
```

Comment:
`AVIF_CONVERSION_AUTO` should be separated from the next line. It should form its own section, separate from the following three sections (libyuv preference, RGB-to-YUV conversion, YUV-to-RGB conversion).

`AVIF_CONVERSION_AUTO` means more than "prefers libyuv if available". It means everything (libyuv preference, chroma downsampling or upsampling filter)  is the default.